src/api/TKRepresentable.h

```diff
@@ -0,0 +1,113 @@
+//
+// Created by Yuhan Niu on 10/16/18.
+// Copyright (c) 2018 Token Inc. All rights reserved.
+//
+
+#import "TKTypedef.h"
+#import "Security.pbobjc.h"
+
+
+/**
+ * Represents the part of a token member that can be accessed through an access token.
+ * 
+ * <p>
+ * The class provides async API
+ * </p>
+ */
+@protocol TKRepresentable
+
+/**
+ * Looks up funding bank accounts linked to Token.
+ *
+ * @param onSuccess callback invoked on success
+ * @param onError callback invoked on error
```

Comment:
keep the param comments in same style

 * param accountId account id
 * param onSuccess callback invoked on success
 * param onError callback invoked on error

Reply:
done